test/library/test_dilation_measurements.py

```diff
@@ -98,16 +98,16 @@ def test_to_sampler_pub(self):
 
         observable = SparsePauliOp(["ZI"], coeffs=[1.0])
         exp_value, std = post_processor.get_expectation_value(observable)
-        self.assertAlmostEqual(exp_value, -0.18749595202757485)
-        self.assertAlmostEqual(std, 0.1428020261876306)
+        self.assertAlmostEqual(exp_value, -0.18749595202757485, places=5)
+        self.assertAlmostEqual(std, 0.1428020261876306, places=5)
         observable = SparsePauliOp(["IZ"], coeffs=[1.0])
         exp_value, std = post_processor.get_expectation_value(observable)
-        self.assertAlmostEqual(exp_value, -0.18749696469140917)
-        self.assertAlmostEqual(std, 0.14280207155875035)
+        self.assertAlmostEqual(exp_value, -0.18749696469140917, places=5)
+        self.assertAlmostEqual(std, 0.14280207155875035, places=5)
         observable = SparsePauliOp(["XI"], coeffs=[1.0])
         exp_value, std = post_processor.get_expectation_value(observable)
-        self.assertAlmostEqual(exp_value, 1.1269484774399672)
-        self.assertAlmostEqual(std, 0.16627566403440813)
+        self.assertAlmostEqual(exp_value, 0.8949297904387811, places=5)
+        self.assertAlmostEqual(std, 0.16958103337866423, places=5)
```

Comment:
We should revisit how our unittests like this one work to avoid dependence on changes to the transpiler pipeline